vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericWaitSteps.java

```diff
@@ -132,6 +137,47 @@ public String toString()
         });
     }
 
+    /**
+     * Waits for an <b><i>element</i></b> with the specified <b>locator</b> to stop moving
+     * @param locator locator to locate element
+     */
+    @When("I wait until element located by `$locator` stops moving")
+    public void waitForElementStopsMoving(Locator locator)
+    {
+        uiContext.getOptionalSearchContext().ifPresent(context -> {
+            locator.getSearchParameters().setWaitForElement(false);
+            IExpectedSearchContextCondition<Boolean> condition = new IExpectedSearchContextCondition<>() {
+                private final List<Point> points = new ArrayList<>();
```

Comment:
why is list needed? can't we keep only last position?

Reply:
yes, if we don't want to see checked coordinates

Reply:
you can log checked locations